mitosheet/mitosheet/utils.py

```diff
@@ -141,6 +143,32 @@ def dfs_to_array_for_json(
     return new_array
 
 
+def add_columns_to_condition_formats(
```

Comment:
I think a more informative name would be something like `get_conditional_formats_objects_to-add_to_excel` -- add is a bit overloaded in this namespace / PR.